### ISSUE TITLE
Classes.dex not found

### ISSUE THREAD

**Issue by scto** [Deenu143/CodeAssist-ALPHA-0.2.9#14]
Build new project from template
Hit build
Error: see Screenshot 
![Screenshot_20231001-010049](https://github.com/Deenu143/CodeAssist-ALPHA-0.2.9/assets/1441369/5f3a669a-5444-4b42-ab67-f0917c5b46c1)

**Comment by Deenu143** [comment]
@scto Don't use the latest action currently I was trying to make javac version 17 but it didn't worked

**Comment by scto** [comment]
ok then that was my failure

**Comment by scto** [comment]
could be a problem with javac

**Comment by Deenu143** [comment]
@scto Yeah, javac doesn't compiles java files when setting the source and taget version to 17

**Comment by scto** [comment]
Upgrade javac from androiide?

**Comment by Deenu143** [comment]
Currently project is not maintained

**Comment by scto** [comment]
author integrated javac in androidide source tree check commit history
